### PR TITLE
fixed errors and stuff

### PR DESCRIPTION
**Pull request by p-casals**

### CODE REVIEW

File: src/main/java/frc/robot/commands/ShooterCommand.java [hunk 1]
```diff
@@ -6,28 +6,29 @@
...
-import com.ctre.phoenix.sensors.SensorVelocityMeasPeriod;
 import edu.wpi.first.wpilibj2.command.CommandBase;
-import frc.robot.subsystems.ShooterSubsystem;
+import frc.robot.subsystems.Shooter;
+
+ //TODO: this is unneccesary: we have the suff in robotcontainer.java
 
 public class ShooterCommand extends CommandBase {
-  ShooterSubsystem shooterSubsystem;
+  Shooter Shooter;
```
Comment (LordSquishers) [inline src/main/java/frc/robot/commands/ShooterCommand.java]: please make Shooter Shooter -> Shooter shooter

File: src/main/java/frc/robot/commands/ShooterCommand.java [hunk 2]
```diff
@@ -6,28 +6,29 @@
...
   private double speed;
 
   /**
    * Creates a new ShooterCommand.
    */
-  public ShooterCommand(ShooterSubsystem p, double speed) {
+  public ShooterCommand(Shooter p, double speed) {
     addRequirements(p);
-    shooterSubsystem = p;
+    Shooter = p;
```
Comment (LordSquishers) [inline src/main/java/frc/robot/commands/ShooterCommand.java]: shooter

File: src/main/java/frc/robot/commands/ShooterCommand.java [hunk 3]
```diff
@@ -6,28 +6,29 @@
...
     this.speed = speed;
     // Use addRequirements() here to declare subsystem dependencies.
   }
 
   // Called when the command is initially scheduled.
   @Override
   public void initialize() {
-    shooterSubsystem.setSpeed(speed);
+    Shooter.setSpeed(speed);
   }
```
Comment (LordSquishers) [inline src/main/java/frc/robot/commands/ShooterCommand.java:32]: lower case

File: src/main/java/frc/robot/commands/ShooterCommand.java [hunk 4]
```diff
@@ -38,7 +39,7 @@ public void execute() {
   // Called once the command ends or is interrupted.
   @Override
   public void end(boolean interrupted) {
-    shooterSubsystem.setSpeed(0);
+    Shooter.setSpeed(0);
```
Comment (LordSquishers) [inline src/main/java/frc/robot/commands/ShooterCommand.java]: lower case

**Review by LordSquishers — CHANGES_REQUESTED** [review]

**Comment by p-casals** [comment]
fixed?

**Review by LordSquishers — APPROVED** [review]
thank you

**Pull request merged**